exchange-v2/test/contracts/v2/SimpleTransferManager.sol

```diff
@@ -1,28 +1,43 @@
 // SPDX-License-Identifier: MIT
 
 pragma solidity 0.7.6;
+pragma abicoder v2;
 
-import "../../../contracts/ITransferManager.sol";
+import "@rarible/exchange-interfaces/contracts/ITransferManager.sol";
+import "@rarible/transfer-manager/contracts/TransferExecutor.sol";
 
-abstract contract SimpleTransferManager is ITransferManager {
-    using SafeMathUpgradeable for uint;
+contract SimpleTransferManager is TransferExecutor, ITransferManager {
```

Comment:
there is a warning about receive() instead fallback(), need to decide what we need to use here (i found something here https://stackoverflow.com/questions/59651032/why-does-solidity-suggest-me-to-implement-a-receive-ether-function-when-i-have-a)

Reply:
it's more about real raribleTransferManager but it concernes this too

Reply:
agree, use receive() instead fallback()